src/pybel/constants.py

```diff
@@ -250,6 +250,10 @@ def get_cache_connection() -> str:
 
 #: The key for an internal edge data dictionary for the relation string
 RELATION = 'relation'
+#: The key for negative edges
+NEGATIVE = 'negative'
+#: The tag for negative edges
+NEGATIVE_TAG = '~'
```

Comment:
I like the pandas reference haha

Reply:
In python the `~` is always the unary negation operator, not just in pandas :)